webpack.config.js

```diff
@@ -1,102 +1,96 @@
-/* eslint-env node */
-const path = require("path");
-const webpack = require("webpack");
-const ExtractTextPlugin = require("extract-text-webpack-plugin");
-const extractStyles = new ExtractTextPlugin("main.css");
-const HtmlWebpackPlugin = require("html-webpack-plugin");
-const externalAssets = require("./lib/javascripts/external_assets");
+const path = require('path')
+const CleanWebpackPlugin = require('clean-webpack-plugin')
+const CopyWebpackPlugin = require('copy-webpack-plugin')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const TranslationsPlugin = require('./webpack/translations-plugin')
+const devDependencies = require('./package.json').devDependencies
+
+// this function reads Zendesk Garden npm dependencies from package.json and
+// creates a jsDelivr url
+const zendeskGardenJsDelivrUrl = (function () {
```

Comment:
I know dan cam up with this, butttt... just like with HtmlWebpackPlugin, I feel we are adding unnecessary complexity to the scaffold.

Why can't we do it like we did in our apps so far? Just include it in the css file?

Reply:
One of the benefits of having an app scaffold is that it's a way for us to encourage practices that are more efficient for our purposes. In this case for example, it's a way for us to reduce bandwidth.